### PR TITLE
Update: Add NewExpression support to comma-style

### PR DESCRIPTION
**Pull request by RazerM**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[x] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What rule do you want to change?**

comma-style

**Does this change cause the rule to produce more or fewer warnings?**

more, if new option is enabled

**How will the change be implemented? (New option, new default behavior, etc.)?**

New `exceptions.NewExpression` option, which defaults to true, and can be set to false for the new behaviour.

**Please provide some example code that this change will affect:**

```js
new Date(
  2017
  ,11
  ,7)
```

**What does the rule currently do for this code?**

No errors.

**What will the rule do after it's changed?**

Gives an error that `',' should be placed last.`

**What changes did you make? (Give an overview)**

I added a `NewExpression` exception in the same style as the existing `comma-style` exceptions (`FunctionExpression` etc.)

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

File: docs/rules/comma-style.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ This rule also accepts an additional `exceptions` object:
     * `"ObjectExpression": true` ignores comma style in object literals
     * `"ObjectPattern": true` ignores comma style in object patterns of destructuring
     * `"VariableDeclaration": true` ignores comma style in variable declarations
+    * `"NewExpression": true` ignores comma style in the parameters of constructor expressions
```
Comment (ilyavolodin) [inline docs/rules/comma-style.md]: Could you add an example into `exceptions` section with this option?

Comment (RazerM) [reply docs/rules/comma-style.md]: I could, but the other exceptions don't have examples, should `NewExpression` be different?

**Comment by ilyavolodin** [comment]
@eslint/eslint-team Does anyone else wants to support this change? We need one more 👍  and a champion

**Comment by kaicataldo** [comment]
This seems like a reasonable enhancement to me! We just need a champion now.

**Comment by platinumazure** [comment]
I'll champion this. Marking as accepted.

**Comment by not-an-aardvark** [comment]
Thanks for contributing!

**Pull request merged**